lib/web_ui/lib/src/engine/window.dart

```diff
@@ -73,7 +73,11 @@ base class EngineFlutterView implements ui.FlutterView {
   final ViewConfiguration _viewConfiguration = const ViewConfiguration();
 
   @override
-  void render(ui.Scene scene) => platformDispatcher.render(scene, this);
+  void render(ui.Scene scene, {ui.Size? size}) {
+    assert((size != null && physicalConstraints.isSatisfiedBy(size)) || (size == null && physicalConstraints.isTight));
+    // TODO(goderbauer): Resepct the provided size when "physicalConstraints" are not allways tight. See TODO on "physicalConstraints".
```

Comment:
@ditman Wondering if you'd be up for wiring this (and the other TODO below) through the web engine.

Reply:
@goderbauer no problem! I think I'm going to use tight constraints as if they were a normal "size", and then do some special casing so constraints + size allow for a web-like reflow!